### PR TITLE
feat: add monitoring dashboard for multicloud/aws

### PR DESCRIPTION
**Pull request by leonzz**
Provide sample cloud monitoring dashboard config file to monitor container status in Anthos on AWS clusters.

Sample config file based from Anthos on-prem(VMware) monitoring dashboard [instructions](https://cloud.google.com/anthos/clusters/docs/on-prem/how-to/logging-and-monitoring#dashboards).

### CODE REVIEW

File: aws-logging-monitoring/monitoring/dashboards/pod-status.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
+{
+  "displayName": "CLUSTER_NAME (Anthos cluster on AWS) pod status",
+  "gridLayout": {
+    "columns": 2,
+    "widgets": [
+      {
+        "title": "Container memory usage per container",
```
Comment (rsgowman) [inline aws-logging-monitoring/monitoring/dashboards/pod-status.json:7]: "Container memory usage per container" seems redundant. How about either: "Memory usage per container" or "Container memory usage"?

File: aws-logging-monitoring/monitoring/dashboards/pod-status.json [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            "label": "y1Axis",
+            "scale": "LINEAR"
+          },
+          "chartOptions": {
+            "mode": "COLOR"
+          }
+        }
+      },
+      {
+        "title": "Container CPU usage per container",
```
Comment (rsgowman) [inline aws-logging-monitoring/monitoring/dashboards/pod-status.json:39]: (again)

File: aws-logging-monitoring/monitoring/dashboards/pod-status.json [hunk 3]
```diff
@@ -0,0 +1,137 @@
+{
+  "displayName": "CLUSTER_NAME (Anthos cluster on AWS) pod status",
```
Comment (rsgowman) [inline aws-logging-monitoring/monitoring/dashboards/pod-status.json:2]: > CLUSTER_NAME

I assume we're going to have instructions that will cause this to be replaced. No action required.

Aside: Once they exist, maybe we should add a README.md file somewhere within aws-logging-monitoring that links back to our docs?

File: aws-logging-monitoring/monitoring/dashboards/pod-status.json (hunk 1 above)
Comment (leonzz) [reply aws-logging-monitoring/monitoring/dashboards/pod-status.json]: Done.

File: aws-logging-monitoring/monitoring/dashboards/pod-status.json (hunk 2 above)
Comment (leonzz) [reply aws-logging-monitoring/monitoring/dashboards/pod-status.json]: Done.

File: aws-logging-monitoring/monitoring/dashboards/pod-status.json (hunk 3 above)
Comment (leonzz) [reply aws-logging-monitoring/monitoring/dashboards/pod-status.json:2]: Yes `CLUSTER_NAME` will be replaced in the usage instruction, which is referenced in `aws-logging-monitoring/monitoring/README.md`.

**Pull request merged**